src/Generator.cc

```diff
@@ -56,147 +59,159 @@ void replaceAll(std::string &_src, const std::string &_oldValue,
 }
 
 /////////////////////////////////////////////////
-Generator::Generator(const std::string &/*_name*/)
+std::vector<std::string> getNamespaces(const std::string &_package)
 {
+  std::vector<std::string> result;
+  std::stringstream ss(_package);
+  std::string item;
+  while (getline (ss, item, '.')) {
+    result.push_back (item);
+  }
+  return result;
 }
 
+
 /////////////////////////////////////////////////
-Generator::~Generator()
+Generator::Generator(const std::string &/*_name*/)
 {
 }
 
+/////////////////////////////////////////////////
+Generator::~Generator() = default;
+
 /////////////////////////////////////////////////
 bool Generator::Generate(const FileDescriptor *_file,
```

Comment:
return is not documented in the `Generator.hh`

Reply:
Documented from https://developers.google.com/protocol-buffers/docs/reference/cpp/google.protobuf.compiler.code_generator#CodeGenerator.Generate.details